docs/components/card.md

```diff
@@ -158,6 +158,50 @@ const App = () => (
 );
 ```
 
+## Card with no shadow
+
+Use the `noShadow` property to remove a card's default box shadow.
+
+```html preview
+<sl-card class="card-basic" noShadow> This is just a basic card with no shadow. </sl-card>
+
+<style>
+  .card-basic {
+    max-width: 300px;
+  }
+</style>
+```
+
+```pug slim
+sl-card.card-basic noShadow="true"
```

Comment:
See explanation for `details.md`
```suggestion
sl-card.card-basic noShadow=true
```

Reply:
bah -- i didn't realize committing your suggested code snippt would get rid of your comment! anyway thank you -- i understand now about the `true` vs `"true"`!